src/libs/telemetry.js

```diff
@@ -14,7 +15,7 @@ export default class Telemetry {
     if(event) {
       return Telemetry.send(event)
     } else {
-      console.error('Telemetry event is missing')
+      Error.display('local_store', 'getAllPois', 'error getting pois', {})
```

Comment:
copy/paste fail: this message is not correct ;)

Reply:
oups :(